izadji/src/pages/DonatePage/DonatePage.tsx

```diff
@@ -1,15 +1,16 @@
 import { ReactNode, useEffect, useState } from 'react';
 import donatePageService from "../../services/donatePageService";
+import {MapComponents} from "../../utils/mappers/sharedMapper";
 
 const DonatePage = () => {
     const [components, setComponents] = useState<ReactNode[]>([]);
-    const [workPrograms, setWorkPrograms] = useState({});
 
     useEffect(() => {
         donatePageService.getDonatePage().then((response) => {
             console.log('donate page', response);
```

Comment:
Please, remove console log if it is unnecessary.

Reply:
That page is incomplete, and will be completed in a following PR, can I do it then? @LazarJovic